rhc-ose-ansible/roles/openshift-provision/tasks/prerequisites.yml

```diff
@@ -0,0 +1,36 @@
+---
+- name: "Cleaning yum repositories"
+  command: "yum clean all"
+  
+- name: "Install required packages"
+  yum:
+    name: "{{ item }}"
+    state: latest
+  with_items:
+    - wget
+    - git
+    - net-tools
+    - bind-utils
+    - bridge-utils
+    - bash-completion
+    - atomic-openshift-utils
+    - vim-enhanced
+
+- name: "Update all packages (this can take a very log time)"
+  yum:
+    name: "*"
+    state: latest
+
+- name: "Verify hostname"
```

Comment:
based on the previous conversations, let's take the hostname stuff out of this role.